src/services/charts/useBranchCoverageMeasurements.tsx

```diff
@@ -33,11 +33,10 @@ const GetBranchCoverageMeasurementsSchema = z.object({
     .object({
       repository: z
         .discriminatedUnion('__typename', [
-          z
-            .object({
-              __typename: z.literal('Repository'),
-            })
-            .merge(MeasurementsSchema),
+          z.object({
+            __typename: z.literal('Repository'),
+            coverageAnalytics: MeasurementsSchema.optional(),
```

Comment:
Can we swap this to `nullable`, in GQL there's no concept of `undefined`. So if this returns `null` than the schema parsing will fail.

Reply:
Great catch - thank you!! today I learned!